lib/matobj2.gd

```diff
@@ -562,28 +562,38 @@ DeclareGlobalFunction( "MakeVector" );
 # Some things that fit nowhere else:
 ############################################################################
 
+############################################################################
+##
 ##  <#GAPDoc Label="MatObj_Randomize_Vectors">
 ##  <ManSection>
-##    <Oper Arg="V" Name="Randomize" Label="for IsVectorObj"/>
-##    <Oper Arg="V,Rs" Name="Randomize" Label="for IsVectorObj,IsRandomSources"/>
+##    <Heading>Randomize</Heading>
+##    <Oper Arg="[Rs,]v" Name="Randomize" Label="for a vector object"/>
+##    <Oper Arg="[Rs,]m" Name="Randomize" Label="for a matrix object"/>
 ##    <Description>
-##      Replaces every entry in <A>V</A> with a random one from the base
-##      domain. If given, the random source <A>Rs</A> is used to compute the
-##      random elements. Note that in this case, the random function for the
-##      base domain must support the random source argument.
+##      Replaces every entry in the mutable vector object <A>v</A>
+##      or matrix object <A>m</A>, respectively, with
+##      a random one from the base domain of <A>v</A> or <A>m</A>,
+##      respectively, and returns the argument.
```

Comment:
Note that the documentation in `orb` says that nothing is returned. I prefer your variant, though, it is useful in some situations. However, this is one more thing that needs to be adapted in `recog`, `cvec` and `meataxe64`. (I'll look into doing that right now).